openscenario/openscenario_interpreter/src/syntax/dynamic_constraints.cpp

```diff
@@ -49,6 +51,7 @@ DynamicConstraints::operator traffic_simulator_msgs::msg::DynamicConstraints() c
   dynamic_constraints.max_deceleration = max_deceleration;
   dynamic_constraints.max_deceleration_rate = max_deceleration_rate;
   dynamic_constraints.max_speed = max_speed;
+  dynamic_constraints.max_jerk = max_jerk;
```

Comment:
maxJerk field does not exist in [DynamicConstraints.](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/DynamicConstraints.html)
Please remove it.